src/lib/libast/meson.build

```diff
@@ -21,6 +21,145 @@ if libpath_result.returncode() == 0
     libast_c_args += [ '-DCONF_LIBPATH=' + libpath_output ]
 endif
 
+
+align_bound_code = '''
```

Comment:
This is okay for now but I would be happier if big blobs of C code wasn't embedded in *meson.build* files. We should establish a pattern where these blobs are in freestanding `*.c` files that are referenced by build rules.

Reply:
These tests are large because they are directly taken from ```iffe```. I believe they can be trimmed down in size. We can have a look at it later. I will leave a TODO comment.